packages/app/components/settings/tabs/billing.tsx

```diff
@@ -105,11 +114,9 @@ const HistoryItem = memo(function HistoryItem({
 }: {
   item: PaymentsHistory;
 }) {
-  const handleViewPress = useCallback(() => {
-    for (let link of item.receipts) {
-      Linking.openURL(link);
-    }
-  }, [item]);
+  const handleViewPress = useCallback((link: string) => {
+    Linking.openURL(link);
+  }, []);
```

Comment:
I misunderstood the code before. You are right, useCallback is not needed here and can be inlined

Reply:
DONE!